.github/workflows/cd-infrastructure-pr.yml

```diff
@@ -1,4 +1,4 @@
-name: Infrastructure_cd
+name: Infra - PR
```

Comment:
I'd prefer not to use acronyms where possible. Can we use the word "infrastructure"? Applies throughout. PR is probably OK as that is more commonly used

Reply:
Fair enough. I've full-formed them all.